addons/l10n_ee/data/account_tax_report_data.xml

```diff
@@ -14,7 +14,7 @@
         </field>
         <field name="line_ids">
             <record id="tax_report_line_1" model="account.report.line">
-                <field name="name">1 - Acts and transactions subject to tax at a rate of 20%</field>
+                <field name="name">1 - Acts and transactions subject to tax at a rate of 22%</field>
```

Comment:
This is missing in the translation files ^^

Reply:
hey, sorry, this spec was added yesterday and it was still wip, I should've warned you :see_no_evil: 
fyi, I stumbled upon a bug while working on this and fixed it in a separate commit in the enterprise branch. 
I think it should be fine to review now :crossed_fingers: 
thanks :slightly_smiling_face: